capstone-ui/src/components/ProfileCard/ProfileCard.jsx

```diff
@@ -1,27 +1,26 @@
 import React from "react";
 import "./ProfileCard.css";
 
-class ProfileCard extends Component {
-  render() {
-    return (
-      <div className="profile">
-        <div className="profile-container">
-          <img
-            src={this.props.pictureURL}
-            alt=""
-            height="200px"
-            width="200px"
-          />
-          <h1>
-            <a href={this.props.profileURL} target="_blank">
-              {this.props.firstName} {this.props.lastName}
-            </a>
-          </h1>
-          <h2>{this.props.headline}</h2>
-        </div>
+const ProfileCard = ({
```

Comment:
This component is not being used anywhere yet. Am I correct?
Where are you planning to add it?